contribs/gmf/src/directives/lidarprofile.js

```diff
@@ -0,0 +1,137 @@
+goog.provide('gmf.lidarProfileComponent');
+goog.require('gmf');
+goog.require('ol.geom.LineString');
+
+
+gmf.module.value('gmfLidarProfileTemplateUrl',
+  /**
+     * @param {!angular.JQLite} $element Element.
+     * @param {!angular.Attributes} $attrs Attributes.
+     * @return {string} Template.
+     */
+  ($element, $attrs) => {
+    const templateUrl = $attrs['gmfLidarProfileTemplateUrl'];
+    return templateUrl !== undefined ? templateUrl :
+      `${gmf.baseTemplateUrl}/lidarprofile.html`;
+  });
+
+
+/**
+ * @param {!angular.JQLite} $element Element.
+ * @param {!angular.Attributes} $attrs Attributes.
+ * @param {!function(!angular.JQLite, !angular.Attributes): string} gmfLidarProfileTemplateUrl Template function.
+ * @return {string} Template URL.
+ * @ngInject
+ */
+function gmfLidarProfileTemplateUrl($element, $attrs, gmfLidarProfileTemplateUrl) {
+  return gmfLidarProfileTemplateUrl($element, $attrs);
+}
+
+
+/**
+ * Provide a component that display a lidar profile panel.
```

Comment:
Please, improve the documentation. See: https://github.com/camptocamp/ngeo/blob/2.2/contribs/gmf/src/directives/displayquerygrid.js#L59-L79

(Example and htmlattributes)